README.md

```diff
@@ -371,7 +371,74 @@ sort of stuff with data from thousands of servers and millions of clients!
 
 ZenoDB relies on a schema file (by default `schema.yaml`).
 
-TODO - fill this out
+### Example: How to add a view
+
+Defining a table as a view means that the data for this table comes from another table or view, rather than an input stream.
```

Comment:
@uaalto Yesterday you mentioned that a view is really just a language construct for creating a table whose properties are derived from an existing table. I think that's probably a better way of explaining this.